tests/e2e/cucumber/features/smoke/share.ocis.feature

```diff
@@ -49,7 +59,7 @@ Feature: share
 
 
   Scenario: file
-    When "Alice" logs in
+    Given "Alice" logs in
```

Comment:
why changing a when step to given?

Reply:
log in is not when in my sense. Isn't it because When step should be the action step. This feature file is for share so sharing action should be first when step

Reply:
if it is a given step, then should be written as:
```suggestion
    Given "Alice" is logged in
```

Reply:
In e2e there is not grammar concept so we can use the same step in the given and when.